src/main/java/com/spotinst/sdkjava/model/OceanGKECreateLaunchSpecRequest.java

```diff
@@ -0,0 +1,58 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.spotinst.sdkjava.client.rest.JsonMapper;
+import com.spotinst.sdkjava.model.bl.gcp.OceanGKELaunchSpec;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OceanGKECreateLaunchSpecRequest {
```

Comment:
Didn't understand the usage of the class. please elaborate.

Reply:
This is request class. which holds all request parameters and set to http client for request processing